packages/explat-client/src/internal/experiment-assignment-store.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import type { ExperimentAssignment } from '../types';
+import * as Validations from './validations';
+
+/**
+ * Class to hold ExperimentAssignment state
```

Comment:
"hold ExperimentAssignment state" isn't very descriptive. Maybe "store existing ExperimentAssignments in memory" or something similar?